CHANGELOG.md

```diff
@@ -2,6 +2,75 @@
 
 This document outlines major changes between releases.
 
+## 0.103.0 "Buckminsterfullerene" (20 Oct 2023)
+
+A major 3.6.0-compatible version of NeoGo with new salty ZKP-related
+functionality, a large batch of deprecated APIs removal and an experimental
+genesis block extension. Build Groth-16 circuits, generate proofs and deploy
+verification contracts easily with the new end-to-end ZKP example and `zkpbinding`
+NeoGo API. Setup node roles via the node configuration starting from the genesis
+block without any painful multisignature transaction forming and invoke any
+custom script in the genesis-level transaction with the new `Genesis` protocol
+configuration extension. And don't forget to move from deprecated RPC client
+APIs, as a lot of them have been removed.
+
+New node configuration format is finally adapted and the deprecated configuration
+sections are permanently removed according to the schedule. Pay attention to the
+`SecondsPerBlock` protocol configuration section that was replaced by
+`TimePerBlock`, a set of node and services address- and port- related configuration
+section that were replaced by the new `Addresses` format, direct `UnlockWallet`
+consensus configuration that was replaced by a separate `Consensus` section and
+a set of node-specific protocol configurations that were moved under a separate
+`P2P` section.
+
+This release fixes a set of bugs including a vulnerability introduced by changes
+in the transaction conflicts storage scheme implemented in #3061 (a part of
+0.102.0 release). With the patch presented, it's impossible to uncontrollably
+pollute the storage with malicious conflicting records. 
```

Comment:
Probably a bit more text than I'd expect. But anyway.

Reply:
Is it important, maybe remove this paragraph at all?

Reply:
I mean whole text, but we can keep it as is.